deltacat/utils/ray_utils/concurrency.py

```diff
@@ -104,7 +104,13 @@ def round_robin_options_provider(
         foo.options(**opt).remote()
     ```
     """
-    assert resource_keys, f"No resource keys given to round robin!"
-    resource_key_index = i % len(resource_keys)
-    key = resource_keys[resource_key_index]
-    return {"resources": {key: resource_amount_provider(resource_key_index)}}
+    opts = kwargs.get("pg_config")
+    if opts: #use pg and bundle id for fault-tolerant round-robin
```

Comment:
Nit: space after `#`.

```suggestion
    if opts: # use pg and bundle id for fault-tolerant round-robin
```